src/main/java/view/InputView.java

```diff
@@ -0,0 +1,50 @@
+package view;
+
+import domain.user.Name;
+import domain.user.Player;
+
+import java.math.BigDecimal;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+
+import static java.util.stream.Collectors.toList;
+
+public class InputView {
+    private static Scanner scanner = new Scanner(System.in);
+    private static final String SEPARATOR = ",";
+    private static final String YES = "y";
+    private static final String NO = "n";
+
+    private InputView() {
+    }
+
+    public static List<String> inputNames() {
+        System.out.println("게임에 참여할 사람의 이름을 입력하세요(쉼표 기준으로 분리).");
+        return Arrays.stream(scanner.nextLine().split(SEPARATOR))
+                .map(String::trim)
+                .collect(toList());
+    }
+
+    public static BigDecimal inputBettingMoneyAmount(Name name) {
+        StringBuilder messageBuilder = new StringBuilder(name.getName());
+        messageBuilder.append("의 배팅 금액은?");
+        System.out.println(messageBuilder);
```

Comment:
```suggestion
        System.out.println(name.getName() + "의 배팅 금액은?");
```
지금 같은 경우에는 위와 같이 변경하는 것이 더 좋지않을까요? 🤔
문자열을 빈번하게 더하는 경우에는 StringBuilder 혹은 StringBuffer를 사용할 수도 있겠지만 지금의 경우에는 문자열을 한 번 더하는 것이 끝이기 때문에 굳이 StringBuilder 객체를 생성할 필요가 없다고 생각됩니다. 

아래의 `inputAnswer` 메서드 또한 동일합니다. 
ResultView 클래스에서도 동일한 고민을 해보셨으면 합니다. 😃